web/src/components/Accordion.tsx

```diff
@@ -1,30 +1,46 @@
 import React, { useRef, useEffect, useState, ReactElement } from 'react'
 import styled from 'styled-components'
 
-const AccordionWrapper = styled.div<{ height: number }>`
-  overflow: hidden;
+const AccordionWrapper = styled.div<{ height: number; $overflowVisible: boolean }>`
   transition: height 0.3s ease;
   height: ${props => props.height}px;
   width: 100%;
+  overflow: ${props => (props.$overflowVisible ? 'visible' : 'hidden')};
```

Comment:
Since you animate the height, you don't need to mess around with changing the overflow, just set it to hidden. That also removes the weird jumping behavior at the end of the opening animation.

Reply:
I had to .. because when you open the calendar and select a date it will take a space under it... you can test it out if you want.

Reply:
Ah, damn. In that case, you'll have to find a different way of fixing the jump :)

https://github.com/user-attachments/assets/86db8b3f-0e55-4dfc-ad1a-de1990e8d599